renovate.json

```diff
@@ -1,18 +1,20 @@
 {
   "extends": [
-    "github>openstack-k8s-operators/renovate-config:default.json5"
+    "github>openstack-k8s-operators/renovate-config:stable.json5"
   ],
   "baseBranches": ["main"],
   "useBaseBranchConfig": "merge",
+  "commitMessageExtra": "",
+  "prBodyColumns": ["Package"],
   "packageRules": [
     {
       "matchPackageNames": ["github.com/openstack-k8s-operators/openstack-operator/apis"],
       "enabled": false
     }
   ],
   "postUpgradeTasks": {
-    "commands": ["make gowork", "make tidy", "make manifests generate"],
+    "commands": ["git reset --hard HEAD","make force-bump", "make gowork", "make tidy", "make manifests generate"],
```

Comment:
* git reset is needed to avoid transitive dependencies leaking in from the bump renovate did.
* make force-bump will pull the openstack-k8s-operators related deps from the 18.0-fr1 branch.